staker/challenge_manager.go

```diff
@@ -132,6 +133,7 @@ func NewChallengeManager(
 	if err != nil {
 		return nil, fmt.Errorf("error creating block challenge backend for challenge %v: %w", challengeIndex, err)
 	}
+	backend.SetDebugEspressoIncorrectHeight(val.GetDebugEspressoIncorrectHeight())
```

Comment:
Shouldn't this be conditional?

Reply:
making this conditional would be better for understanding.